rmw_connext_shared_cpp/src/node.cpp

```diff
@@ -53,6 +53,23 @@ create_node(
   // This String_dup is not matched with a String_free because DDS appears to
   // free this automatically.
   participant_qos.participant_name.name = DDS::String_dup(name);
+  // since the participant name is not part of the DDS spec
+  // the node name is also set in the user_data
+  DDS_Long name_length = strlen(name);
+  bool success = participant_qos.user_data.value.length(name_length + 6);
+  if (!success) {
+    RMW_SET_ERROR_MSG("failed to resize participant user_data");
+    return NULL;
+  }
+  const char * prefix = "name=";
```

Comment:
We would reduce the number of hardcoded values if we defined this prefix above and use it's length everywhere below (rather than hardcoding `5`s and `6`s)

Reply:
Done in 95dfd983a9f62769c98a6e9af5b47c0beddf27e7.

Reply:
Done in 221769e5d58d3c778a239a71039226b183063efe.